### ISSUE TITLE
Highlight accessibility statements (output)

### ISSUE THREAD

**Issue by yatil** [w3c/wai-components-gallery#17]
Better Highlight Statements: The accessibility statements provided during the submission process should be clearly identifiable from the front-end, to communicate the status more clearly.
